packages/sdk/src/types.ts

```diff
@@ -104,7 +109,7 @@ export type FtAddresses = {
   usdt: USDT_ADDRESS;
 }
 
-export type RPC_OPTIONS  = 'lava' | 'near' | 'beta'
+export type RPC_OPTIONS  = 'lava' | 'near' | 'beta' | 'fastnear'
```

Comment:
Why is this type duplicated here and in the rpc package?

Reply:
because this was a fix, and I didnt wanted to have to reinstall sdk on rpc package just to have this new value on the type. we have the same problem with mintbase-wallet
https://github.com/Mintbase/mintbase-wallet/pull/1012